apps/src/publicKeyCryptography/cryptographyMath.js

```diff
@@ -0,0 +1,84 @@
+/** @file Utilities for doing cryptography widget calculations */
+import memoize from 'fast-memoize';
+
+export const primesInRange = memoize((n, m) => {
+  let primes = [];
+  if (n <= 2 && m >= 2) {
+    primes.push(2);
+  }
+  for (let i = n; i <= m; i++) {
+    if (isPrime(i)) {
+      primes.push(i);
+    }
+  }
+  return primes;
+});
+
+const isPrime = memoize(n => {
+  const upperDivisor = Math.sqrt(n) + 1;
+  for (let i = 2; i < upperDivisor; i++) {
+    if (n % i === 0) {
+      return false;
+    }
+  }
+  return true;
+});
+
+export const privateKeyList = memoize(publicModulus => {
+  if (!publicModulus) {
+    return [];
+  }
+  let potentialVals = [];
+  let privateKeys = [];
+  for (let i = 0; i < publicModulus - 2; i++){
+    potentialVals.push(i);
+  }
+  potentialVals[1] = 0;
+  for (let j = 0; j < publicModulus - 2; j++) {
+    // let pct = parseInt(publicModulus*0.05);
```

Comment:
Hmm, probably not.  I lifted this from the original widget code, and identified this as a dead code path.  I'll remove it now that I'm confident that it's not needed.